packages/back-end/src/controllers/admin.ts

```diff
@@ -80,6 +114,107 @@ export async function putOrganization(
     updates.ownerEmail = ownerEmail;
     orig.ownerEmail = org.ownerEmail;
   }
+  if (verifiedDomain) {
+    updates.verifiedDomain = verifiedDomain;
+    orig.verifiedDomain = org.verifiedDomain;
+  }
+  if (autoApproveMembers !== org.autoApproveMembers) {
+    updates.autoApproveMembers = autoApproveMembers;
+    orig.autoApproveMembers = org.autoApproveMembers;
+  }
+  if (enterprise !== org.enterprise) {
+    updates.enterprise = enterprise;
+    orig.enterprise = org.enterprise;
+  }
+  if (freeSeats !== org.freeSeats) {
+    updates.freeSeats = freeSeats;
+    orig.freeSeats = org.freeSeats;
+  }
+
+  await updateOrganization(org.id, updates);
+
+  await req.audit({
+    event: "organization.update",
+    entity: {
+      object: "organization",
+      id: org.id,
+    },
+    details: auditDetailsUpdate(orig, updates),
+  });
+
+  return res.status(200).json({
+    status: 200,
+  });
+}
+
+// delete organization - For now, we're just marking the organization as deleted
+export async function disableOrganization(
+  req: AuthRequest<{ orgId: string }>,
+  res: Response
+) {
+  if (!req.superAdmin) {
+    return res.status(403).json({
+      status: 403,
+      message: "Only superAdmins can disable organizations",
+    });
+  }
+
+  const updates: Partial<OrganizationInterface> = {};
+  const orig: Partial<OrganizationInterface> = {};
+  const { orgId } = req.body;
+  const org = await getOrganizationById(orgId);
+
+  if (!org) {
+    return res.status(404).json({
+      status: 404,
+      message: "Organization not found",
+    });
+  }
+
+  updates.disabled = true;
+  orig.disabled = org.disabled;
+
+  await updateOrganization(org.id, updates);
+
+  await req.audit({
+    event: "organization.update",
```

Comment:
We might consider having a more explicit event type:
`event: "organization.disable",`

Reply:
And `organization.enable` below...

Reply:
done